static/ankersrv.js

```diff
@@ -119,35 +141,30 @@ $(function () {
 
         _open() {
             $(this.badge).removeClass("text-bg-success text-bg-danger").addClass("text-bg-warning");
-            if (this.open)
-                this.open(this.ws);
+            if (this.open) this.open(this.ws);
```

Comment:
Are all these formatting changes deliberate, or an artifact of vscode?

If you'd like it to look like this, we can talk about it.

...but if it's just the computer having an opinion, please cut it out :stuck_out_tongue:

Reply:
I'll go through and clean all of the formatting changes up. I also added in some config to `.vscode/settings.json` to force vscode not to auto format anything but what was modified

Reply:
Sounds good :)